.infrastructure/clusterIp.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: todoapp-clusterip
+  namespace: todoapp
+spec:
+  selector:
+    app: todoapp
+  ports:
+  - protocol: TCP
+    port: 80
+    targetPort: 8080
+  type: ClusterIP
```

Comment:
Add Eof(End-of-file), need to add a blank line at the end
![image](https://github.com/user-attachments/assets/aecb4f16-13e7-4636-8d2f-64933ec0fdc5)